core/arch/x86/instr_create.h

```diff
@@ -1396,6 +1396,27 @@
     instr_create_1dst_1src((dc), OP_vphsubwd, (d), (s))
 #define INSTR_CREATE_vphsubdq(dc, d, s) \
     instr_create_1dst_1src((dc), OP_vphsubdq, (d), (s))
+/* AVX-512 VEX */
```

Comment:
nit: Better to append instead of inserting here in between AVX and AVX2.

Reply:
Done.